tests/conversion

```diff
@@ -0,0 +1,264 @@
+#!/bin/sh
+set -eu
+
+# Source helper functions.
+. ./bin/helpers
+
+install_deps jq unzip
+
+# Install LXD
+install_lxd
+
```

Comment:
`set -x` here so that we can trace the important bits. Helps a lot when reading the logs.